src/components/oculus-touch-controls.js

```diff
@@ -191,6 +192,18 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
 
     // Offset pivot point
     controllerObject3D.position = PIVOT_OFFSET;
+
+    // Fix issue #2965 for default Touch model, rotate it 45 degrees.
```

Comment:
No need to reference issues on source comments. The reference in the commit is good enough